blocks/form/maskingUtil.js

```diff
@@ -0,0 +1,114 @@
+const masking = {
```

Comment:
It make sense to put it in side components/mask itself

Reply:
Could you please confirm if there have been any changes to this code compared to the version currently running on the site?